recipes-support/skl/files/0001-iommu-command-buffer-restored.patch

```diff
@@ -15,13 +15,13 @@ index 7502079..39222c4 100644
 +++ b/include/iommu.h
```

Comment:
We should not need SKL patches in this layer, we should rely on upstream tb repo

Reply:
@macpijan done in [1927317](https://github.com/3mdeb/meta-trenchboot/pull/28/commits/19273177e3ba6311b82834e423852ab4399750c3)